components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,54 @@
+﻿import {apiTryLogin} from "../../api/loginApiClient.module";
+import React, {FormEvent, useState} from "react";
+import scss from "./LoginForm.module.scss";
+
+interface LoginFormProps {
+    userId: string;
+    password: string;
+    setUserId: Function;
+    setPassword: Function;
+}
+
+type FormStatus = "READY" | "FINISHED";
+
+export function LoginForm(props: LoginFormProps): JSX.Element {
+
+    const [status, setStatus] = useState<FormStatus>("READY");
+
+    function tryLogin(event: FormEvent): void {
+        event.preventDefault();
```

Comment:
What happens if we leave this out?

Reply:
I assume nothing as the default behaviour of submitting a form is what we want anyway?

Reply:
Cool, as long as it works.

Reply:
OK I had another look, it stops the page refreshing if the submit fails, so will probably be useful so the user doesn't have to type in their ID again if they used the wrong password. You happy for me to add it back in?

Reply:
Ah, okay. Yeah, go for it.